test_programs/compile_success_empty/unquote_struct/src/main.nr

```diff
@@ -10,11 +10,13 @@ fn foo(x: Field, y: u32) -> u32 {
 
 // Given a function, wrap its parameters in a struct definition
 comptime fn output_struct(f: FunctionDefinition) -> Quoted {
-    let fields = f.parameters().map(|param: (Quoted, Type)| {
+    let fields = f.parameters().map(
+        |param: (Quoted, Type)| {
```

Comment:
Hmm wonder how this got merged in my PR without nargo fmt rejecting it.

Also interesting seeing all the blue now that Noir is a recognized language on github.